acnportal/acnsim/network/current.py

```diff
@@ -0,0 +1,60 @@
+import cmath
+import math
+from copy import deepcopy
+import numpy as np
+import pandas as pd
+
+class Current(pd.Series):
+    """ A simple representation of currents as an extension of pandas Series.
+    Includes addition, subtraction, and multiplication (by scalar) operators.
+
+    Attributes:
+        loads (Dict[str, number]): Dictionary which maps a load_id to its coefficient in the aggregate current.
+
+    Args:
+        loads (Dict[str, number], str, or List[str]): If dict, a dictionary mapping load_ids to coefficients. If str a
+            load_id. If list, a list of load_ids. Default None. If None, loads will begin as an empty dict.
+    """
+    def __init__(self, loads=None):
+        # Backwards compatibility with previous Current specification methods
+        if isinstance(loads, dict):
+            super().__init__(loads)
+        elif isinstance(loads, str):
+            super().__init__({loads : 1})
+        elif loads is None:
```

Comment:
Move elif loads is None to the end of the cases to make it more clear that this is a default case.